src/course/the2nd/Task2.java

```diff
@@ -0,0 +1,14 @@
+package course.the2nd;
+
+public class Task2 {
+	public static void main(String[] args) {
+		int sum = 0;
+		for(int i = 1; i <= 10; i++) {
+			
+			sum += i;	
```

Comment:
余分な改行は消しましょう！
EclipseやIntelliJに自動フォーマット機能があるので導入してみてください！
`Eclipse format`
`Eclipse save actions`
などででます！
```suggestion
			sum += i;	
```

Reply:
他のファイルにも横展開して対応いただけるとありがたいです！